.github/workflows/ci.yml

```diff
@@ -121,14 +124,14 @@ jobs:
         run: ./scripts/build.sh /tmp/e2e-test/avalanchego/plugins/srEXiWaHuhNyGwPUi444Tu47ZEDwxTWrbQiuD7FmgSAQ6X7Dy
       - name: Run Warp E2E Tests
         shell: bash
-        run: AVALANCHEGO_BUILD_PATH=/tmp/e2e-test/avalanchego DATA_DIR=/tmp/e2e-test/warp-data ./scripts/run_ginkgo_warp.sh
-      - name: Upload Artifact
-        if: always()
+        run: AVALANCHEGO_BUILD_PATH=/tmp/e2e-test/avalanchego ./scripts/run_ginkgo_warp.sh
+      - name: Upload tmpnet network dir for warp testing
         uses: actions/upload-artifact@v3
+        if: always()
         with:
-          name: subnet-evm-e2e-logs-warp
-          path: /tmp/network-runner-root-data*/
-          retention-days: 5
```

Comment:
should we keep the retention-days?

Reply:
Just to avoid keeping things indefinitely. I was unaware of the 90 day default, which seems fine.